util/Migrator/DbScripts/2023-12-04_00_GrantIndexes.sql

```diff
@@ -0,0 +1,47 @@
+IF COL_LENGTH('[dbo].[Grant]', 'Id') IS NULL
+BEGIN
+    ALTER TABLE [dbo].[Grant]
+        ADD [Id] INT NOT NULL IDENTITY
+
+    ALTER TABLE [dbo].[Grant]
+        DROP CONSTRAINT [PK_Grant];
+
+    ALTER TABLE [dbo].[Grant]
+        ADD CONSTRAINT [PK_Grant] PRIMARY KEY CLUSTERED ([Id] ASC);
+
+    CREATE UNIQUE INDEX [IX_Grant_Key]
+        ON [dbo].[Grant]([Key]);
+END
+GO
+
+IF EXISTS(SELECT *
+FROM sys.views
+WHERE [Name] = 'GrantView')
+    BEGIN
+    DROP VIEW [dbo].[GrantView];
+END
+GO
+
+CREATE VIEW [dbo].[GrantView]
+AS
+    SELECT
+        *
+    FROM
+        [dbo].[Grant]
+GO
+
```

Comment:
Would be better to drop these first so that the PK rebuild does not trigger a rebuild of these